testing/jfrog-cli/APKBUILD

```diff
@@ -1,16 +1,16 @@
 # Maintainer: Gennady Feldman <gena01@gmail.com>
-pkgname=jfrog-cli-go
-_pkgname=jfrog-cli
-pkgver=1.29.0
+pkgname=jfrog-cli
+pkgver=1.29.1
 pkgrel=0
 pkgdesc="JFrog cli"
 url="https://jfrog.com/getcli/"
 arch="all"
 license="Apache-2.0"
 makedepends="go"
 options="!check"
+replaces="jfrog-cli-go"
 source="$pkgname-$pkgver.tar.gz::https://github.com/jfrog/$pkgname/archive/$pkgver.tar.gz"
-builddir="$srcdir/$_pkgname-$pkgver"
+builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
You can remove this now since builddir already defaults to `builddir="$srcdir/$pkgname-$pkgver"`.

Reply:
Fixed